development/using-windows.md

```diff
@@ -10,66 +10,94 @@ Installation instructions are mostly the same as they written in [the README](ht
 
 ## Installing Ubuntu in the Windows Subsystem for Linux.
 
-For the rest of this document we're going to presume that you're using Ubuntu. Medic probably works all on all distributions, but Ubuntu is likely the best supported.
+For the rest of this document we're going to presume that you're using Ubuntu (18.04) in WSL. Medic probably works all on all distributions, but Ubuntu is likely the best supported.
 
 First, follow Microsoft's [instructions on enabling and installing linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10). At the end of this process you should have a linux terminal.
 
 Note: for the rest of this tutorial **in linux** means code executing or performing actions in the subsystem command prompt, while **in Windows** means code executing or performing actions in Windows natively.
 
 ## CouchDB
 
-As of writing CouchDB wouldn't autostart due to systemd not existing (I think?), and wasn't manually starting due to erlang errors.
+As of writing CouchDB wouldn't autostart (due to systemd not existing?), and wasn't manually starting due to erlang errors.
 
 Luckily, there is a perfectly working CouchDB installation for Windows:
  - Download from [CouchDB](https://couchdb.apache.org/#download) and install the Windows version. This will create a Windows service.
  - Run it either by directly executing `C:\CouchDB\bin\couchdb.cmd` or by starting the service
 
 Then go to `http://localhost:5984/_utils/#/setup` in Windows and do the single node setup. Once done head back to linux and confirm it works:
 
-```
-scdf@PC:/mnt/c/Users/stefa/Code/medic$ curl http://localhost:5984/
+```bash
+$: curl http://localhost:5984/
 {"couchdb":"Welcome","version":"2.3.1","git_sha":"c298091a4","uuid":"5f60350abaaa11c0131a5630e83ae979","features":["pluggable-storage-engines","scheduler"],"vendor":{"name":"The Apache Software Foundation"}}
 ```
 
 ## Installing NPM
+Start your WSL instance (Ubuntu), not WSL as they take you to two different defult directories. 
 
 The default `npm` in linux is really old and doesn't have `npm ci`, which we need.
 
 Instead use [nvm](https://github.com/nvm-sh/nvm) to install a later version.
```

Comment:
May be just `Instead use [nvm](https://github.com/nvm-sh/nvm) to install "nvm install 11.3".` and then remove the paragraph after? I don't think it's adding much.